linux/user-and-file-management/manipulating-environment-variables/adding-directories-to-your-path.md

```diff
@@ -18,68 +10,64 @@ type: normal
 category: must-know
 
 tags:
-
   - path
-
   - linux
-
   - terminal
-
   - bash
-
   - bashrc
-
   - export
-
   - introduction
 
-
 ---
 
-# Adding directories to your `$PATH`
+# Adding Directories to `PATH`
 
 ---
 ## Content
 
-**PATH** is an environmental variable that informs the shell within which directories to search for executable files in response to commands issued by a user.
+To do this, edit the `.bashrc` file in the home directory and append the following line (chaining the existing `PATH` at the end):
 
-To do this, edit the `.bashrc` file in the home directory and append the following line (chaining the existing path additions) :
-```
+```bash
 export PATH=/path/to/dir:$PATH
 ```
-You can open *.bashrc* in terminal by running:
-```
-$ nano ~/.bashrc
+
+You can open `.bashrc` in terminal by running:
+
+```bash
+nano ~/.bashrc
 # ~/ being the implicit path for
 # your home folder
 ```
 
-For example, to add the directory */usr/sbin*, the following would be used:
-```
+For example, to add the directory `/usr/sbin`, the following would be used:
```

Comment:
we should explain what this command does. it pre-pends a new path to the previous one?

Reply:
yes. we explain how it works 2 insights before this